addons/mail/static/src/messaging/entity/attachment_viewer/attachment_viewer.js

```diff
@@ -0,0 +1,75 @@
+odoo.define('mail.messaging.entity.AttachmentViewer', function (require) {
+'use strict';
+
+const { registerNewEntity } = require('mail.messaging.entity.core');
+
+function AttachmentViewerFactory({ Entity }) {
+
+    class AttachmentViewer extends Entity {
+
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @override
+         * @param {mail.messaging.entity.Attachment} [data.attachment]
+         * @param {mail.messaging.entity.Attachment[]} [data.attachments]
```

Comment:
Missing params doc: angle, isImageLoading, scale

Doc of params should be here and not inline.